packages/cli/lib/projects.js

```diff
@@ -60,7 +61,7 @@ const writeProjectConfig = (configPath, config) => {
     fs.writeFileSync(configPath, JSON.stringify(config, null, 2));
     logger.debug(`Wrote project config at ${configPath}`);
   } catch (e) {
-    logger.error(`Could not write project config at ${configPath}`);
+    logErrorInstance(e);
```

Comment:
hmm, it might be nice to log out the nice-to-read message by default. I would actually be ok with keeping this error as it was

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1115/commits/4045486aa0478e4b985436849d5418aba6119cc3